common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1439,7 +1439,15 @@ def load_block_type(self, block_type):
         """
         Returns a subclass of :class:`.XBlock` that corresponds to the specified `block_type`.
         """
-        if block_type in self.disabled_xblock_types:
+        if hasattr(self, "_services") and 'request_cache' in self._services:
```

Comment:
@ormsbee Here is my first attempt to use a request cache. It works (when running the LMS-- I haven't looked into how I might write a test for it), but there has to be a more elegant way.

FYI @cpennington, as perhaps you can direct me on how to get ahold of the request_cache in a better way.